files/6_2_7_include_folders.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+grep -E -v '^(halt|sync|shutdown)' /etc/passwd | awk -F: '($7 != "'"$(which nologin)"'" && $7 != "/bin/false") { print $1 " " $6 }' | while read user dir; do
+    if [ ! -d "$dir" ]; then
+        echo "The home directory ($dir) of user $user does not exist."
+    else
+        for file in $dir/.[A-Za-z0-9]*; do
+            if [ ! -h "$file" -a "$file" ]; then
```

Comment:
This line in the regular script is
`if [ ! -h "$file" -a -f "$file" ]; then`
So it only checks for files